core/src/test/scala/org/apache/spark/deploy/client/AppClientSuite.scala

```diff
@@ -245,8 +249,8 @@ class AppClientSuite
       execRemovedList.add(id)
     }
 
-    def executorDecommissioned(id: String, message: String): Unit = {
-      execDecommissionedList.add(id)
+    def executorDecommissioned(id: String, decommissionInfo: ExecutorDecommissionInfo): Unit = {
+      execDecommissionedMap.putIfAbsent(id, decommissionInfo)
```

Comment:
If we get a new decommission message would we want to overwrite the old one?

Reply:
Yeah I should probably have stronger checks here considering that this is a test.